mapper.go

```diff
@@ -24,10 +24,11 @@ import (
 )
 
 var (
-	identifierRE = `[a-zA-Z_][a-zA-Z0-9_]+`
-	metricLineRE = regexp.MustCompile(`^(\*\.|` + identifierRE + `\.)+(\*|` + identifierRE + `)$`)
-	labelLineRE  = regexp.MustCompile(`^(` + identifierRE + `)\s*=\s*"(.*)"$`)
-	metricNameRE = regexp.MustCompile(`^` + identifierRE + `$`)
+	identifierRE      = `[a-zA-Z_][a-zA-Z0-9_]*`
+	metricLineRE      = regexp.MustCompile(`^(\*\.|` + identifierRE + `\.)+(\*|` + identifierRE + `)$`)
```

Comment:
I found comments that full utf-8 might even be possible - it seems this isn't standardised. I'll update when I get home.